packages/filecoin-actor-utils/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  preset: 'ts-jest',
+  testRegex: '.(spec|test).ts$',
+  testEnvironment: 'node',
+  collectCoverageFrom: ['**/*.{ts,js}', '!**/node_modules/**'],
+  testPathIgnorePatterns: ['/node_modules/'],
+  moduleNameMapper: {
+    '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy'
```

Comment:
I know this exists everywhere else, but probably unnecessary here..

Reply:
Yeah good call!